kafka-ui-react-app/src/components/Schemas/New/__test__/New.spec.tsx

```diff
@@ -21,4 +24,20 @@ describe('New Component', () => {
   it('renders component', () => {
     expect(screen.getByText('Create new schema')).toBeInTheDocument();
   });
+  it('submit button will be disabled while form fields are not filled', () => {
+    const submitBtn = screen.getByRole('button', { name: /submit/i });
+    expect(submitBtn).toBeDisabled();
+  });
+  it('submit button will be enabled when form fields are filled', async () => {
+    const subject = screen.getByPlaceholderText('Schema Name');
+    const schema = screen.getAllByRole('textbox')[1];
+    await act(() => {
+      userEvent.type(subject, subjectValue);
+    });
+    await act(() => {
+      userEvent.type(schema, schemaValue);
+    });
+    const submitBtn = screen.getByRole('button', { name: /submit/i });
+    expect(submitBtn).toBeEnabled();
+  });
```

Comment:
@rAzizbekyan pls take a look if this warning is valid

Reply:
@workshur I checked it and locally it's not showing error like this

Reply:
All added cases generate warnings. pls fix it
```
console.error
    Warning: An update to New inside a test was not wrapped in act(...).

    When testing, code that causes React state updates should be wrapped into act(...):

    act(() => {
      /* fire events that update state */
    });
    /* assert on the output */

    This ensures that you're testing the behavior the user would see in the browser. Learn more at https://reactjs.org/link/wrap-tests-with-act
        at New (/Users/workshur/repos/kafka-ui/kafka-ui-react-app/src/components/Schemas/New/New.tsx:31:27)
        at Routes (/Users/workshur/repos/kafka-ui/kafka-ui-react-app/node_modules/.pnpm/packages/react-router/lib/components.tsx:253:3)
        at WithRoute (/Users/workshur/repos/kafka-ui/kafka-ui-react-app/src/lib/testHelpers.tsx:33:55)
        at Router (/Users/workshur/repos/kafka-ui/kafka-ui-react-app/node_modules/.pnpm/packages/react-router/lib/components.tsx:173:13)
        at MemoryRouter (/Users/workshur/repos/kafka-ui/kafka-ui-react-app/node_modules/.pnpm/packages/react-router/lib/components.tsx:33:3)
        at QueryClientProvider (/Users/workshur/repos/kafka-ui/kafka-ui-react-app/node_modules/.pnpm/@tanstack+react-query@4.0.5_ef5jwxihqo6n7gxfmzogljlgcm/node_modules/@tanstack/react-query/src/QueryClientProvider.tsx:71:3)
        at TestQueryClientProvider (/Users/workshur/repos/kafka-ui/kafka-ui-react-app/src/lib/testHelpers.tsx:42:3)
        at Provider (/Users/workshur/repos/kafka-ui/kafka-ui-react-app/node_modules/.pnpm/react-redux@8.0.2_nfqigfgwurfoimtkde74cji6ga/node_modules/react-redux/lib/components/Provider.js:19:3)
        at Object.<anonymous>.exports.ThemeProvider (/Users/workshur/repos/kafka-ui/kafka-ui-react-app/node_modules/.pnpm/styled-components@5.3.1_uuaz5p7xzfmtjacf6iqf7idnby/node_modules/styled-components/src/models/ServerStyleSheet.js:68:13)
        at AllTheProviders (/Users/workshur/repos/kafka-ui/kafka-ui-react-app/src/lib/testHelpers.tsx:67:5)
```

Reply:
I fixed it all, I'm not sure what's the problem that it still fails as when I run tests locally everything is fine

Reply:
The issue is fixed